siliconcompiler/report/streamlit_viewer.py

```diff
@@ -195,7 +198,11 @@ def show_file_preview(header_col_width=0.89):
             is given to the download button.
     """
     path = streamlit.session_state['selected'][0]
-    file_name, file_extension = os.path.splitext(path)
+    # This file extension may be '.gz', if it is, it is compressed.
```

Comment:
Could you either add a test case for the compressed file viewer or a screenshot of it in action to this pull request?